daemon/emer-boot-id-provider.c

```diff
@@ -105,18 +104,16 @@ emer_boot_id_provider_get_property (GObject    *object,
       break;
 
     default:
-      G_OBJECT_WARN_INVALID_PROPERTY_ID (object, property_id, pspec);
+      G_OBJECT_WARN_INVALID_PROPERTY_ID (self, property_id, pspec);
```

Comment:
This should now have a cast: `G_OBJECT (self)`.

My personal preference would be to have this function's signature take a `GObject` since `get_property` is an override of a `GObject` vfunc, but that's up to you.